pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -17,6 +17,8 @@
 , moreutils
 , cacert
 , nodePackages
+, speechd
+, withTTS ? false
```

Comment:
why not make this enabled by default? i'd imagine it's configurable in the app, no?

otherwise lgtm!

Reply:
> If it is enabled here by default, should I make a PR to so it is the same for the Discord package?

IMO, TTS, as an accessibility feature, should be enabled by default so that people don't have to hunt it down to enable it (which is made much harder if you actually rely on TTS to navigate the system, for example). People looking to "debloat" things by removing optional dependencies required for some part of the app to function can disable it manually.